scripts/monitors/check_wsgi.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python2.7
```

Comment:
/usr/bin/env python will get us the correct python version.

We aren't using this script anywhere today for monitoring so I think we can just delete it.